dev/test/specs/starter-test.js

```diff
@@ -78,12 +78,25 @@ function loadSeleniumUtils(cdnBranch) {
     script.src = `${cdnBranch}/test/selenium-utils.js`;
     document.getElementsByTagName('head')[0].appendChild(script);
   }, cdnBranch);
+  browser.waitUntil(() => {
+    try {
+      const result = browser.execute('pierceShadows(["head"])');
+    } catch (e) {
+      console.log(
+        `caught an exception looking for pierceShadows(); we'll try again. Text: ${e}`
```

Comment:
How many times do we try again? Is the retry handled by browser.waitUntil? Presumably at some point we'll stop trying so I wondered whether at that point this message would read incorrectly.